internal/command/views/test.go

```diff
@@ -223,14 +224,18 @@ func (t *TestHuman) DestroySummary(diags tfdiags.Diagnostics, run *moduletest.Ru
 	t.Diagnostics(run, file, diags)
 
 	if state.HasManagedResourceInstanceObjects() {
-		t.view.streams.Eprint(format.WordWrap(fmt.Sprintf("\nOpenTofu left the following resources in state after executing %s, and they need to be cleaned up manually:\n", identifier), t.view.errorColumns()))
+		t.view.streams.Eprint(format.WordWrap(fmt.Sprintf("\nOpenTofu left the following resources in state after executing %s, the left-over resources can be cleaned using the statefile written to disk(errored_test.tfstate) or can be done manually:\n", identifier), t.view.errorColumns()))
```

Comment:
tiny tiny tiny nitpick:

```suggestion
		t.view.streams.Eprint(format.WordWrap(fmt.Sprintf("\nOpenTofu left the following resources in state after executing %s, the left-over resources can be managed using the statefile written to disk(errored_test.tfstate) or can be done manually:\n", identifier), t.view.errorColumns()))
```

Reply:
(same for the other instances of this error message)

Reply:
done 👍🏼